modules/widgets/contact-info.php

```diff
@@ -42,8 +42,22 @@ function __construct() {
 		 * Enqueue scripts and styles.
 		 */
 		public function enqueue_scripts() {
+			$google_url = 'https://maps.googleapis.com/maps/api/js?sensor=false';
+			/**
+			 * Set a Google Maps API Key.
+			 *
+			 * @since 4.1.0
+			 *
+			 * @param string $key Google Maps API Key
+			 */
+			$key = apply_filters( 'jetpack_google_maps_api_key', null );
+
+			if ( ! empty( $key ) ) {
+					$google_url = add_query_arg( 'key', $key, $google_url );
+				}
+
 			wp_enqueue_script( 'jquery' );
-			wp_enqueue_script( 'google-maps', 'https://maps.googleapis.com/maps/api/js?sensor=false' );
+			wp_enqueue_script( 'google-maps', esc_url( $google_url ) );
```

Comment:
`add_query_arg` doesn't provide any protection and since it is an user-provided value, we should. But, you're right, I'm using `esc_url` in the default `display` context. Changing it to send null context will save the ampersands.